TLM/TLM/Util/PriorityRoad.cs

```diff
@@ -141,6 +249,40 @@ public static class PriorityRoad {
             }
         }
 
+        /// <summary>
+        /// Quick-setups the given junction as priority junction.
+        /// The roads on the segmentList are considered as main road.
+        /// All other roads are considered as minor road.
+        /// Also detects:
+        ///  - semi-roundabout
+        ///  - split avenue into 2 oneway roads (only applicable to first or last node).
+        /// </summary>
+        public static void FixHighPrioirtyJunction(ushort nodeId, List<ushort> segmentList) {
+            if (nodeId == 0) {
+                return;
+            }
+
+            List<ushort> nodeSegments = new List<ushort>();
+            for (int i = 0; i < 8; ++i) {
+                ushort segId = nodeId.ToNode().GetSegment(i);
+                if (segId != 0) {
+                    bool main = segmentList.Contains(segId);
+                    if (main) {
+                        nodeSegments.Insert(0, segId);
+                    } else {
+                        nodeSegments.Add(segId);
+                    }
+                }
+            }
+
+            if (nodeSegments.Count < 3) {
+                Log._Debug("FixJunction: This is not a junction.");
```

Comment:
Who will read this debug message? User will never see it, developer will not understand it out of context. 
Remove before merge or convert into a proper error with code location.

Reply:
Lets say you are trying to debug why HighPrio-ir-ity (lol) quick setup does not work. if you see in debug log "FixJunction: This is not a junction." it serves as very useful information. if you don't see this message it is also useful.
I can add nodeID.